### ISSUE TITLE
`prefer-math-min-max`: ignore Date objects

### ISSUE THREAD

**Issue by kkmuffme** [sindresorhus/eslint-plugin-unicorn#2900]
Same issue reported for bigints https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2509 also exists Date objects.
It creates invalid code at best, and creates security issues at worst when the date object comparison is used for auth-related stuff.

```js
const now = new Date();
const later = new Date(now.getTime() + Math.ceil(Math.random() * 1000 - Math.random() * 1000) );
const a = now < later ? 'a' : 'b';
```
